build.psm1

```diff
@@ -293,8 +293,6 @@ function Start-PSBuild {
                      "fedora.24-x64",
                      "win7-x64",
                      "win7-x86",
-                     "win81-x64",
-                     "win10-x64",
```

Comment:
Even though `win81-x64` and `win10-x64` are removed from this list, when running `Start-PSBuild` we will still see the publish folder path is `netcoreapp2.0\win10-x64\` on a win10 machines and `netcoreapp2.0\win81-x64\` on a win81 machine. However, we are not allowed to specify `win10-x64` or `win81-x64` for the `-Runtime` parameter here. The user experience seems broken to me.

My opinion is to keep `Start-PSBuild` and `New-PSOptions` unchanged, and only make some changes to the packaging scripts.